internal/config/config.go

```diff
@@ -59,6 +76,7 @@ func ParseConfig() *Config {
 		ClusterDomain:       *clusterDomain,
 		Username:            *userName,
 		InventoryAddress:    *inventoryAddress,
+		MetricsPort:         *metricsPort,
```

Comment:
is this `metricsPort` in use later in code at all?

Reply:
Nope, removed and replaced with a `disable-metrics` flag like the other feature flags